### PR TITLE
Can now drag plasma tanks/parts into frames/collectors

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting your Pull Request

Can now drag plasma tanks/parts into frames (borgs too)

# Wiki Documentation

n/a

# Changelog

:cl:  
rscadd: Can now drag plasma tanks/parts into frames/collectors.
/:cl:

### CODE REVIEW

**Comment by adamsong** [comment]
Power creep

**Comment by ShatteredParadox** [comment]
finally

**Comment by maxion12345** [comment]
Borg's shouldnt be able to build complex machines without the RPED upgrade to prevent power creep

File: code/game/machinery/constructable_frame.dm [hunk 1]
```diff
@@ -280,7 +280,8 @@
 	..()
 
 /obj/structure/frame/machine/MouseDrop_T(atom/dropping, mob/user)
-	if(istype(dropping, /obj/item/circuitboard))
+	if(istype(dropping, /obj/item/circuitboard) || istype(dropping, /obj/item/stock_parts))
```
Comment (adamsong) [inline code/game/machinery/constructable_frame.dm:283]: This just makes the RPED upgrade much less useful

**Review by adamsong — APPROVED** [review]

**Pull request merged**